src/v/net/probes.cc

```diff
@@ -292,7 +292,7 @@ void tls_certificate_probe::loaded(
         return;
     }
 
-    auto to_tls_serial = [](bytes_view b) {
+    auto to_tls_serial = [](const auto& b) {
```

Comment:
nit, perhaps this could be `std::span<std::byte>`

Reply:
tangential (sorry):
1. Did my [recent seastar change](https://github.com/redpanda-data/seastar/pull/113) break this?
2. Would it make sense to expedite [this](https://github.com/redpanda-data/redpanda/pull/18287) instead? (stalled last week on CI troubles, but I can push)

Reply:
1. Yes I believe so
2. I am happy with either approach